docs/user-guide/creating-application/git-material.md

```diff
@@ -12,41 +12,49 @@ Devtron also supports multiple git repositories in a single deployment. We will
 
 ## 1. Select Provider
 
-In the provider section, you have to select the git provider of your code repository like- Github, Gitlab, and Bitbucket, etc.You can configure your git provider via [global configuration](../global-configurations/).
+In the provider section, you have to select the git provider of your code repository like- Github, Gitlab, and Bitbucket, etc. If you are using a private git repository, you can configure your git provider via [global configuration](../global-configurations/).
```

Comment:
instead of `[global configuration](../global-configurations/).` point user to `git provider config`